honeybee_vtk/config.py

```diff
@@ -208,114 +231,84 @@ def _validate_data(data: DataConfig, model: Model) -> bool:
     Returns:
         A boolean value.
     """
-    # if file_paths is empty
-    if not data.file_paths:
-        raise ValueError(
-            f'For object with name {data.name} There are not file paths'
-            ' provided to load data from.'
+    grids_info_json = pathlib.Path(data.path).joinpath('grids_info.json')
+    # TODO: Confirm with Chris if this check for json is needed. I think it's not.
```

Comment:
Yes, I don't think you don't need this check here. The grids_info.json will always be a valid JSON file

Reply:
Removed